.github/workflows/ci-macos.yml

```diff
@@ -0,0 +1,44 @@
+name: CI-macos
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    name: "Python ${{ matrix.python-version }}"
+    runs-on: macos-latest
+
+    strategy:
+      matrix:
+        python-version: [2.7, 3.6, 3.7, 3.8]
```

Comment:
I would make Linux/Mac OSX one file and have the `runs-on` be part of the matrix. (Unless that's not supported or something?)

Reply:
It is supported, and I do plan on doing it in the future. Since I'm not super experienced with Github actions, I'm just trying to keep things simple for now until I get more comfortable.

Also, when we build the wheels, I'm not sure how much code will be similar, so this saves me from having to split them back up in that case.